src/Tabs/_TabContent.js

```diff
@@ -0,0 +1,36 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const TabContent = (props) => {
+    const { children, id, selected, className, ...rest } = props;
+
+    // css classes for tab panels
+    const tabPanelClasses = classnames(
+        'fd-tabs__panel',
+        className
+    );
+
+    return (
+        <div
+            {...rest}
+            aria-expanded={selected}
+            className={tabPanelClasses}
+            id={id}
+            role='tabpanel' >
+            {children}
+        </div >
+    );
+};
+TabContent.displayName = 'TabContent';
+
+TabContent.propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    id: PropTypes.string,
```

Comment:
The `id` prop has no way of getting set since this is a non-exported component.  The way to do it would be to include `id` in the `tabContentProps` object, but then that would end up on the `<div>` by virtue of the `...rest` spread.  The `className` prop is in a similar situation, however, that can stay since you need to have a way to get the `fd-tabs__panel` class on the `<div>`.